### PR TITLE
Fix it so we don't stop after the first error/file

### PR DESCRIPTION
**Pull request by duglin**
Also made the verbose setting more consistent across the tools

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -1470,7 +1470,7 @@ Platforms SHOULD initiate orphan mitigation in the following scenarios:
 | 201 with malformed response | Failure | Yes |
 | All other 2xx | Failure | Yes |
 | 408 | Timeout failure | Yes |
-| All other 4xx | Service Broker rejected request | No |
+| All other 4xx | Request rejected | No |
```
Comment (duglin) [inline spec.md]: I made this change just so the checker doesn't need to look for "5 word" columns to ignore, that's getting kind of long. Plus saying "Service Broker" isn't really necessary, its clear who is doing the rejection.

**Comment by avade** [comment]
LGTM

File: tools/verify-phrases.sh [hunk 2]
```diff
@@ -164,18 +168,11 @@ function checkFile {
     for phrase in "${bannedPhrases[@]}"; do
       if [[ "${words[@]^^} " == "${phrase^^} "* ]]; then
       ll=${words[*]}
-      echo line ${lines[0]}: \'${ll:0:${#phrase}}\' is banned | tee -a $err
+      echo line ${lines[0]}: \'${ll:0:${#phrase}}\' is banne
```
Comment (mattmcneeney) [inline tools/verify-phrases.sh]: missing a `d`

Comment (duglin) [reply tools/verify-phrases.sh]: fixed!

**Comment by duglin** [comment]
Actually, I've purposely tried to keep the scripts separate in order to keep them maintainable. One large script that tries to do it all might make it harder for people to understand what's going on - or just scare people. Most of the scripts have the same boiler-plate logic for flag parsing, etc... and so its really just the "checkFile" type of function people need to focus on.

**Comment by angarg12** [comment]
LGTM

**Comment by mattmcneeney** [comment]
LGTM (@avade need yours again btw)

**Comment by duglin** [comment]
2 more reviews needed

**Comment by fmui** [comment]
LGTM

**Comment by avade** [comment]
LGTM

**Pull request merged**